tests/fixtures/concat/concat-typescript.twig

```diff
@@ -1,9 +1,9 @@
 <div class="{{ class|default('') }}" style="{{ style|default('') }}">
   <div style="{{ 'fontSize: ' ~ size ~ 'px' }};"></div>
-  <div>{{'foo' ~ ' bar'}}</div>
-  <div>{{text ~ ' foo'}}</div>
-  <div>{{text ~ 1}}</div>
-  <div>{{size ~ ' foo'}}</div>
-  <div>{{size + 1}}</div>
-  <div>{{1 + 1}}</div>
+  <div>{{ 'foo' ~ ' bar' }}</div>
```

Comment:
Didn't you just recently made it so there are no spaces inbetween curly brackets? 😲 
But I actually like the space there.